### ISSUE TITLE
Typo in LANDUSE.TBL

### ISSUE THREAD

**Issue by RobWatersIEA** [wrf-model/WRF#830]
Category 14 in the winter section is missing a comma in the first column in both the MODIS and MODIFIED_IGBP_MODIS_NOAH.

I know a small one but thought best to post regardless...

> WINTER
> 
> 1,     12.,   .60,   .95,   50.,    5.,  3.00, 29.2e5, 'Evergreen Needleleaf Forest'
> 2,     12.,   .50,   .95,   50.,    5.,  1.67, 29.2e5, 'Evergreen Broadleaf Forest'
> 3,     15.,   .60,   .93,   50.,    5.,  2.60, 25.0e5, 'Deciduous Needleleaf Forest'
> 4,     17.,   .60,   .93,   50.,    5.,  2.40, 25.0e5, 'Deciduous Broadleaf Forest'
> 5,     14.,   .60,   .93,   20.,    6.,  1.12, 41.8e5, 'Mixed Forests'
> 6,     22.,   .20,   .93,    1.,    4.,  1.30, 20.8e5, 'Closed Shrublands'
> 7,     22.,   .25,   .93,    1.,    4.,  1.24, 20.8e5, 'Open Shrublands'
> 8,     22.,   .20,   .93,    1.,    4.,  1.30, 20.8e5, 'Woody Savannas'
> 9,     20.,   .15,   .92,   15.,    3.,  2.00, 25.0e5, 'Savannas'
> 10,    23.,   .30,   .92,   10.,    4.,  2.00, 20.8e5, 'Grasslands'
> 11,    14.,   .725,  .95,   30.,    6.,  1.32, 35.5e5, 'Permanent wetlands'
> 12,    20.,   .60,   .92,    5.,    4.,  2.00, 25.0e5, 'Croplands'
> 13,    15.,   .10,   .88,   80.,    3.,  1.67, 18.9e5, 'Urban and Built-Up'
> 14     20.,   .40,   .92,    5.,    4.,  2.00, 25.0e5, 'cropland/natural vegetation mosaic'
> 15,    70.,   .95,   .95,   0.1,    5.,    0., 9.0e25, 'Snow and Ice'
> 16,    23.,   .05,   .90,    1.,    2.,  0.81, 12.0e5, 'Barren or Sparsely Vegetated'
> 17,     8.,   1.0,   .98,  0.01,    6.,    0., 9.0e25, 'Water'
> 18, 15., .60, .93, 30., 5., 1.75, 9.0e25, 'Wooded Tundra'

**Comment by davegill** [comment]
@smileMchen @kkeene44 @weiwangncar 
This is on the release_v4.1 branch. 
The incorrect file is `run/LANDUSE.TBL`
Both MODIS and MODIFIED_IGBP_MODIS_NOAH, in the winter section have category 14 missing an initial comma.
This does not impact results, but should be fixed.

@RobWatersIEA 
Rob,
Thanks for pointing this out.

1. There should be no impact of the missing comma. Since this is list directed input, either white space or a comma would be suitable for parsing the individual fields.

2. I pulled out the read portion of the routine that handles this file to verify this assertion. I ran the code for both MODIS and MODIFIED_IGBP_MODIS_NOAH, and with the original run/LANDUSE.TBL and a corrected file. I get no diffs in the printed standard out.
```
program emulate

character(len=80) :: lutype,mminlu
logical :: found_lu

integer :: LUCATS,LUSEAS, lun, nsn, ls, li

real :: ALBD,SLMO,SFEM, SFZ0,THERIN,SCFX,SFHC

mminlu = ' '
mminlu(1:5)='MODIS'
mminlu(1:24)='MODIFIED_IGBP_MODIS_NOAH'

OPEN(landuse_unit, FILE='../run/LANDUSE.TBL',FORM='FORMATTED',STATUS='OLD',IOSTAT=ierr)
REWIND(landuse_unit)

 1999 CONTINUE
        READ (landuse_unit,*,END=2002)LUTYPE
        print *,lutype
        GOTO 2003
 2002   CONTINUE
        print *,'INPUT FILE FOR LANDUSE REACHED END OF FILE' 
        stop
 2003   CONTINUE
        READ (landuse_unit,*)LUCATS,LUSEAS
        print *,LUCATS,LUSEAS

        FOUND_LU = LUTYPE.EQ.MMINLU
        IF(FOUND_LU)THEN
          LUN=LUCATS
          NSN=LUSEAS
        ENDIF
        DO ls=1,luseas
            READ (landuse_unit,*)
          DO LC=1,LUCATS
            IF(found_lu)THEN
                READ (landuse_unit,*)LI,ALBD,SLMO,SFEM,        &
                           SFZ0,THERIN,SCFX,SFHC
                print *,LI,ALBD,SLMO,SFEM,        &
                           SFZ0,THERIN,SCFX,SFHC
            ELSE
                READ (landuse_unit,*)
            ENDIF
          ENDDO
        ENDDO
        IF(NSN.EQ.1.AND.FOUND_LU) THEN
           ISN = 1
        END IF

      print *,'not using ',trim(lutype)
      IF(.NOT. found_lu ) GOTO 1999

end program
```

**Comment by RobWatersIEA** [comment]
Thanks @davegill :) 

I was just poking around and noticed it. There wasn't any unexpected behavior that got me to it.

**Comment by davegill** [comment]
@smileMchen @kkeene44 @weiwangncar 
Folks,
As a reminder (to me also), we should fix this in v4.1.1, if we have not already. If does not change results.

**Comment by weiwangncar** [comment]
Repo code changed in 4.1.1.
